src/settings/Settings.tsx

```diff
@@ -662,7 +662,7 @@ export const SETTINGS: { [setting: string]: ISetting } = {
     },
     "custom_themes": {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
-        default: [],
+        default: false,
```

Comment:
```suggestion
        default: [] as ICustomTheme[],
```

and export ICustomTheme from `theme.ts` and add its `name: string` field which seems to be missing